Tutorial1-1Basics/src/main/java/com/smarttoolfactory/tutorial1_1basics/tutorial_list/TutorialList.kt

```diff
@@ -1064,6 +1065,23 @@ fun createLayoutTutorialList(): List<TutorialSectionModel> {
         tagColor = LayoutListColor
     )
 
+    val tutorial3_7_4 = TutorialSectionModel(
```

Comment:
I think it would be better to put this amazing example to Graphics section rather than Layouts.